src/lib/app.ts

```diff
@@ -66,10 +65,6 @@ export default async function getApp(
     app.use(baseUriPath, favicon(path.join(publicFolder, 'favicon.ico')));
     app.use(baseUriPath, express.static(publicFolder, { index: false }));
 
-    if (config.enableOAS) {
-        app.use(`${baseUriPath}/oas`, express.static('docs/api/oas'));
```

Comment:
Should we put the removal in a separate PR?

Reply:
Let's, I'll revert this.